### ISSUE TITLE
Problem with Administrator and user

### ISSUE THREAD

**Issue by FrancescoDL99** [cschneegans/unattend-generator#16]
Spectacular tool. Works great, although I'm finding a little problem. I'm trying to create a file that sets up 2 users account. One administrator and one normal user. Once the installation of windows is finished, it always creates only a single user without permissions. I also tried to create an autounattend file that sets a single user as administrator, but in the end it creates one with the right name, but it's local and without permissions.

**Comment by cschneegans** [comment]
Well, that should not happen. Please attach the `autounattend.xml` file you are using so I can have a look.

**Comment by FrancescoDL99** [comment]
This is the last try i have made with the single user account.
[autounattend.txt](https://github.com/user-attachments/files/15988530/autounattend.txt)
Is .txt, i can not attach .xml file here. I don't know how to do it :)

**Comment by cschneegans** [comment]
I did not recognize a problem until I tried to elevate a process and saw this strange dialog:

![image](https://github.com/cschneegans/unattend-generator/assets/30263434/3818af72-b3f9-45ed-a495-b5569fe6bc8b)

It turns out that the sole user account was not a member of any group. I noticed that in your `autounattend.xml` file the user account name is the same as the computer name. Apparently, Windows does not like this.

Change either the user account name or the computer name. I will add additional checks to prevent this in the future. Thank you for bringing this to my attention!
